app/views/content/subjects/physics/_article.html.erb

```diff
@@ -0,0 +1,198 @@
+    </p>
+  </section>
+   <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "It's inspiring getting 'wow, that's how it works' from pupils as they grasp a subject that they thought was hard.",
+    name: "Jane Doran, physics and robotics teacher from Coventry, West Midlands",
+    background: "grey",
+    large: true
+  ) %>
+</section>
+
+ <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithOneImageComponent.new(
+      heading: "Next steps into teacher training",
+      colour: "yellow",
+      image_path: "static/content/hero-images/0006.jpg",
+      image_alt: "A class of pupils sitting at desks looking at the camera.",
+    ) %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Help to become a physics teacher</h3>
+
+    <p>
+      Our experienced teacher training advisers can guide you through everything you need to know to apply for physics teacher training, free of charge. They can help you to:
+    </p>
+    <ul>
+      <li>understand more about physics courses</li>
+      <li>answer questions about fees and funding</li>
+      <li>make an application for a physics course</li>
+      <li>prepare for your teacher training interview</li>
+    </ul>
+    <p><a href="/teacher-training-advisers">Find out about advisers</a></p>
+
+    <p>
+      You can also <a href="https://www.ogdentrust.com/teacher-support/training-to-be-a-teacher/">find out more about becoming a physics teacher from the Ogden Trust</a>, a charitable trust which promotes the teaching and learning of physics.
+    </p>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Do you need a physics degree?</h3>
+    <p>
+      To train to teach physics in a school in England, you'll need a bachelor's degree, but it does not have to be in physics.
+    </p>
+
+    <p>
+      In 2022, candidates with degrees in subjects as varied as physiotherapy, psychology, and zoology were offered places on physics teacher training courses.
+    </p>
+
+    <p>
+      Having relevant A levels can also show your subject knowledge, if you do not have a degree in physics.
+    </p>
+
+    <p>
+      <a href="/is-teaching-right-for-me/qualifications-you-need-to-teach">Find out about the qualifications needed to teach</a>.
+    </p>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Experience school life</h3>
+
+    <p>
+      If you're studying for a degree and interested in teaching physics, you could find out more about applying for <a href="/teaching-internship-providers">a paid teaching internship</a>. The 3-week programme can help you to understand what it's really like in the classroom and get a feel for school life.
+    </p>
+
+    <p>
+      Alternatively, if you're not currently studying for a degree, you can still <a href="/train-to-be-a-teacher/get-school-experience">get school experience</a> to find out what it's like to be in a classroom.
```

Comment:
Is this slightly misleading? If you are currently studying for a degree, you can go GSE but this sentence makes it sound like you can't?

Also - do we need `Alternatively` here?